Makefile

```diff
@@ -0,0 +1,75 @@
+USERNAME=vrutkovs
```

Comment:
Removing this line will allow the default `env` var of the logged in user (USERNAME) to be used.  This can still be overridden at the command line should you desire a different cluster name.

Reply:
good idea, implemented that